README.md

```diff
@@ -1,75 +1,137 @@
-## Django Framework Skeleton for Google App Engine
+## Python Django Skeleton for Google App Engine
 
 A skeleton for building Python applications on Google App Engine with the
-[Django framework](https://www.djangoproject.com/), using [djangoappengine](http://djangoappengine.readthedocs.org/en/latest/).
-
-This skeleton is a bare-bones "hello world" application.
+[Django Web Framework](https://www.djangoproject.com/) using 
+[Cloud SQL](https://cloud.google.com/sql/docs/introduction).
 
 See our other [Google Cloud Platform github
 repos](https://github.com/GoogleCloudPlatform) for sample applications and
-scaffolding for other Python frameworks and use cases.
+scaffolding for other python frameworks and use cases.
+
+Note that this project no longer uses the built-in Django library but
+uses `pip -t` to vendor Django 1.8 into the lib/ folder. 
+
+This project is based on the introductory [Django Tutorial](https://docs.djangoproject.com/en/1.8/intro/tutorial01/)
+polls applications.
+
+## Project Creation
+
+The basic skeleton of this project was created using the standard Django commands:
+
+   ```
+   django-admin startproject mysite
+   ```
+
+The DATABASE field in the myproject/settings.py is modified to use CloudSQL locally, and through
```

Comment:
Make this a list, and also mention the creation of `app.yaml` and `appengine_config.py` and what they do.